Library/Homebrew/version.rb

```diff
@@ -340,8 +340,8 @@ def self._parse(spec)
     m = /-((?:\d+\.)*\d+(?:[abc]|rc|RC)\d*)$/.match(stem)
     return m.captures.first unless m.nil?
 
-    # e.g. foobar-4.5.0-beta1, or foobar-4.50-beta
-    m = /-((?:\d+\.)*\d+-beta\d*)$/.match(stem)
+    # e.g. foobar-4.5.0-alpha5, foobar-4.5.0-beta1, or foobar-4.50-beta
+    m = /-((?:\d+\.)*\d+-(alpha|beta|rc)\d*)$/.match(stem)
```

Comment:
Might want to use a non-capturing group here to be consistent with everything else, but I want to know why they're used in the first place.